patterns/filters.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Title: WooCommerce Filters
+ * Slug: woocommerce-blocks/all-filters
```

Comment:
Not sure about the title or slug, any suggestions/recommendations? 🤔

Reply:
I think its confusing because "filters" mean multiple things in the context of WordPress & WooCommerce. Perhaps one of the following?

* `WooCommerce Product Filters` | `woocommerce-blocks/product-filters`
* `WooCommerce Default Product Filters` | `woocommerce-blocks/default-product-filters`